Simulation/ThrustModel.m

```diff
@@ -5,27 +5,45 @@
 %https://www.electricrcaircraftguy.com/2014/04/propeller-static-dynamic-thrust-equation-background.html
 %This function returns an array of Thrust in the x and y directions
 
-function Thrust = ThrustModel(StateVector, Throttle)
-  %constants we get from propeller 
-  diameter = 6; %inches
-  pitch = 3; %inches (how far forward propeller moves in one rotation
-  Kv = 2280; %RPM per volt
-  Max_Volt = 11.1; %volts
-  
+function [Thrust, B_dot] = ThrustModel(SimData)
+  %constants we get from propeller
+  diameter = 6; % Inches
+  pitch = 3; % Inches (how far forward propeller moves in one rotation)
+  Kv = 2280; % RPM per volt
+  Max_Volt = 4.2; % Volts
+  Min_Volt = 3.2; % Volts
+  Avg_Volt = (Max_Volt + Min_Volt)/2; % Volts
+
   %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
   %Note that Kv*Max_Volt = Max Theoretical RPM.
   %To scale down this RPM, we will use the 'Throttle' input, which is a percentage
-  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-  
-  %calculated values
-  airspeed = sqrt(StateVector(3)^2 + StateVector(4)^2);
-  RPM = Max_Volt*Kv*Throttle;%use Throttle to scale down the max RPM 
-  Thrust = 4.392399*10^(-8)*RPM*diameter^(3.5)*pitch^(-.5)*((4.23333*10^(-4))*RPM*pitch-airspeed)%calculate thrust (based on model in excel file)
-  
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+  %%How to get Throttle:
+  battery_cap_percent = SimData.Plane.Cap/SimData.Plane.MaxCap;
```

Comment:
`SimData.Cap` should probably be a more descriptive name like `SimData.BatteryCap` since "cap" could mean a lot of things.